### PR TITLE
Create personal-bio-site-1.md

### PR DESCRIPTION
**Pull request by morecallan**

### CODE REVIEW

**Comment by rousell** [comment]
I think this looks good! :+1

File: personal-bio-site-1.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+1. Each place you've worked must be contained in an article element.
+1. Each article element should have a header that contains the name of the company and the dates of your employment. 
+1. Each article should contain at least one section that describes your position there.
+1. No footer required for these articles.
+
+#### Contact information page
+
+1. One article.
+1. A heading (e.g. "How to contact me").
+1. Four sections that contain a home address, a phone number, and two social media URLs.
```
Comment (zoeames) [inline personal-bio-site-1.md]: lets not have them push up their address and phone number.  maybe email instead?  also bonus points if they do their social media links

Comment (morecallan) [inline personal-bio-site-1.md]: Agree. God, what has NSS been making us do this whole time???

File: personal-bio-site-1.md [hunk 2]
```diff
@@ -0,0 +1,51 @@
+# Personal Bio Site
+
+## Setup
+
+Setting up your project the correct way will enable you to have a hosted website. 
+
+##### Create a GitHub repository with the name 'YOUR-GITHUB-USERNAME.github.io' (make sure to replace 'YOUR-GITHUB-USERNAME' with your ACTUAL username - ditto for the following steps). 
+
+```bash
```
Comment (zoeames) [inline personal-bio-site-1.md]: go ahead and remove this block of code.  Today I want us to demo how to set this project up.  I think we will use the opportunity to show them how to initialize with a readme

File: personal-bio-site-1.md [hunk 3]
```diff
@@ -0,0 +1,51 @@
+# Personal Bio Site
+
+## Setup
+
+Setting up your project the correct way will enable you to have a hosted website. 
+
+##### Create a GitHub repository with the name 'YOUR-GITHUB-USERNAME.github.io' (make sure to replace 'YOUR-GITHUB-USERNAME' with your ACTUAL username - ditto for the following steps).
```
Comment (zoeames) [inline personal-bio-site-1.md]: leave this in here

**Pull request merged**